com.unity.cinemachine/Samples~/3D Samples/Clearshot/LeadShot.unity

```diff
@@ -0,0 +1,3309 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!29 &1
+OcclusionCullingSettings:
```

Comment:
Other things to tweak in tis scene:
- Put a nonzero MinDuration in the ClearShot, to avoid hysteresis
- Increase the min speed of the runners, so they don't walk so much